src/sentry/static/sentry/app/actionCreators/modal.tsx

```diff
@@ -50,10 +50,7 @@ export async function openSudo({onClose, ...args}: OpenSudoModalOptions = {}) {
   );
   const {default: Modal} = mod;
 
-  openModal(deps => <Modal {...deps} {...args} />, {
-    modalClassName: 'sudo-modal',
```

Comment:
Was this class name unused?

Reply:
Yes this was totally unused afaict, checked here and gs